### ISSUE TITLE
Set delay input box's border has insufficient contrast against its adjacent color in both focused and non-focused states

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#876]
The following AI-windows mode(s) have this issue:
- Regular mode
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![23 1](https://user-images.githubusercontent.com/22068434/88357145-19dfd080-cd1f-11ea-8741-00d9586d4854.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this will be in build 1.1.1369.1. Please regress and close once the build is out. Thanks!

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1380.01. I will file a separate issue for the system default behavior i.e focused input box's border in light mode not having sufficient contrast against its adjacent color
